File: Answers/05_if/e05_2_showTheBiggerNumber.cs
using System;

class Program
{
    static void Main()
    {
        int a = int.Parse(Console.ReadLine());
        int b = int.Parse(Console.ReadLine());
        int c = int.Parse(Console.ReadLine());

        if (a >= b && a >= c)
        {
            Console.WriteLine("a is bigger than b and c");
        }
        else if (b >= a && b >= c)
        {
            Console.WriteLine("b is bigger than a and c");
        }
        else
        {
            Console.WriteLine("c is bigger than a and b");
        }
    }
}

File: Answers/05_if/e05_2_showTheBiggerNumber.py
a = int(input("Give a: "))
b = int(input("Give b: "))
c = int(input("Give c: "))

if a >= b and a >= c:
    print("a is bigger than b and c")
elif b >= a and b >= c:
    print("b is bigger than a and c")
else:
    print("c is bigger than a and b")